keyboards/gmmk/pro/config.h

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2021 Glorious, LLC <salman@pcgamingrace.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define DEVICE_VER      0x0001
+#define VENDOR_ID       0xFEED
```

Comment:
Didn't you guys want to support VIA? Because these IDs need to be changed to make VIA work (and other changes, too).
https://caniusevia.com/docs/configuring_qmk#change-vendor_id-product_id

Reply:
Hey @Pascamel ! yes, we wish to do VIA eventually as well. Right now we are just concerned with getting QMK support fully. 

I will get the data for these attributes ASAP. Is there anything else we need to do for QMK to be approved for merge?

Reply:
Hi,
(btw you quoted the wrong person)
I am not part of QMK but just an enthusiast that stumbled upon this PR :) Looking forward to the GMMK Pro

Reply:
@Pascamel Awesome!! Love that you just hopped in. Yes, we do definitely plan on that as well. Just want to get in the main support before launch :)

Reply:
Thanks!
If there are still some features missing when I get mine, I will be happy to help :)